kernel/aster-nix/src/fs/procfs/pid/cmdline.rs

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use super::*;
```

Comment:
I remembered that we want to avoid using `use super::*` except in `mod test`? But we do have some `use super::*` in a couple of files in `procfs/pid/`. Not sure what policy is being enforced here.

Reply:
Thanks. So I need some input here ... Right now I just leave the `use super::*` as it is.

Reply:
cc @tatetian

Reply:
> I remembered that we want to avoid using use super::* except in mod test

Your memory has served your well 😄

Reply:
Thanks @lrh2000 ! The `use super::*` has been replaced with 
```rust
use crate::{
    fs::{
        procfs::template::{FileOps, ProcFileBuilder},
        utils::Inode,
    },
    prelude::*,
    Process,
};
```
Also, I replaced the `use super::*` in `procfs/self_.rs`, `procfs/pid/fd.rs`, `procfs/pid/exe.rs`, `procfs/pid/comm.rs` with certain modules.